### PR TITLE
Reflect some comments

### PR DESCRIPTION
**Pull request by Hoverbear**
Signed-off-by: Ana Hobden <operator@hoverbear.org>

### CODE REVIEW

File: text/2019-03-04-hibernate-raft.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ This RFC proposes to hibernate a peer when it reaches a stable state.
 
 There are several ticks when driving a peer. The cost of tick is usually
 small. However, when there are a lot of regions in one TiKV instances, the
-cost can't be negligible. We have observed that an idle cluster with many
-regions can waste a lot of CPUs to send useless heartbeats and tick to routine
-checks. Even in an online cluster with tens of thousands of regions, enlarge
+cost can be significant. We have observed that an idle cluster with many
+regions can waste a lot of CPU time to send useless heartbeats and ticks for routine
```
Comment (BusyJay) [inline text/2019-03-04-hibernate-raft.md]: I think it should be "tick" as it's behind "to".

File: text/2019-03-04-hibernate-raft.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ Following describes how to disable all these ticks when possible.
 
 ### Raft
 
-Raft tick is used to driving a raft state machine, so leader can send
-heartbeats, follower can campaign when leader is down. To disable it, we need
-to make sure raft can still work properly. I suggest adding following rules to
+Raft tick is used to drive a raft state machine, so a leader can send
+heartbeats, and a follower can campaign when the leader is down. To disable it,
+to make sure raft can still work properly. The following rules are needed to
```
Comment (BusyJay) [inline text/2019-03-04-hibernate-raft.md]: Isn't it weird to begin with “to” in both clauses?

File: text/2019-03-04-hibernate-raft.md [hunk 3]
```diff
@@ -52,44 +52,44 @@ make it work:
...
    `steady` to true.
 
-8. If a follower finds out network to leader is unavailable, it marks `steady`
+8. If a follower finds out the network to leader is unavailable, it marks `steady`
    to false and starts ticking.
 
 When a cluster is started, because of 3, all peers are ticking. It works
-just the old way. Because of 2, ticks of followers will not stop until a
-leader is elected and at lease a new log is appended. If there is no further
+same as the old way. Because of 2, ticks of followers will not stop until a
```
Comment (BusyJay) [inline text/2019-03-04-hibernate-raft.md]: Is "the" necessary for "same as"?

File: text/2019-03-04-hibernate-raft.md (hunk 1 above)
Comment (Hoverbear) [reply text/2019-03-04-hibernate-raft.md]: You mean "useless heartbeats and tick for"?

File: text/2019-03-04-hibernate-raft.md (hunk 2 above)
Comment (Hoverbear) [reply text/2019-03-04-hibernate-raft.md]: Yes I think I accidentally deleted something unintended. I'll resolve this.

File: text/2019-03-04-hibernate-raft.md (hunk 3 above)
Comment (Hoverbear) [reply text/2019-03-04-hibernate-raft.md]: Yes!

**Comment by Hoverbear** [comment]
@BusyJay PTAL again

File: text/2019-03-04-hibernate-raft.md (hunk 1 above)
Comment (BusyJay) [reply text/2019-03-04-hibernate-raft.md]: "send heartbeats and tick for", ticks will never be sent out.

File: text/2019-03-04-hibernate-raft.md [hunk 4]
```diff
@@ -8,14 +8,14 @@ This RFC proposes to hibernate a peer when it reaches a stable state.
...
-cost can't be negligible. We have observed that an idle cluster with many
-regions can waste a lot of CPUs to send useless heartbeats and tick to routine
-checks. Even in an online cluster with tens of thousands of regions, enlarge
-the ticks' interval can usually lead to better performance. Threaded raftstore
-can ease the problem but can't solve it. In general, not all regions keep
-working all the time. Unbalanced load is a very common use case in practice.
-For those idle regions, we can hibernate them to avoid resources waste, hence
-get a better performance.
+cost can be significant. We have observed that an idle cluster with many
+regions can waste a lot of CPU time to send useless heartbeats and ticks for
```
Comment (BusyJay) [inline text/2019-03-04-hibernate-raft.md:12]: Any updates?

**Comment by BusyJay** [comment]
Since there is no updates for a long time, I'm going to merge this and fix the wrong description in other following commits.

**Pull request merged**